src/deluge/model/clip/instrument_clip.h

```diff
@@ -76,6 +76,8 @@ class InstrumentClip final : public Clip {
 	void seeWhatNotesWithinOctaveArePresent(bool[], int32_t, Song* song, bool deleteEmptyNoteRows = true);
 	void transpose(int32_t, ModelStackWithTimelineCounter* modelStack);
 	void nudgeNotesVertically(int32_t, ModelStackWithTimelineCounter* modelStack);
+	int32_t semitonesNudged;
```

Comment:
This doesn't belong here - it's almost always 0, and even when it's not it doesn't contain useful information about the clip. The value is tied to the action of turning the knob so the storage location should be too. Use a static variable in the function instead